### DISCUSSION TITLE
How to prevent a specific application to focus?

### DISCUSSION THREAD

**Discussion by antoyo** [i3/i3#5553] [Q&A]
Hi.
I'd like to prevent the app Discord to ever take the focus.
I tried the following:
```
no_focus [title="Discord"]
focus_on_window_activation none
```
It works, but it also prevent other applications from focusing.
So how can I make this work for Discord only?
Thanks.

**Comment by arthfl** [comment]
This https://i3wm.org/docs/userguide.html#focus_on_window_activation might be what you want.

**Reply by antoyo** [reply]
I'm not sure how this is different than the config I posted above. It seems this applies to all applications, but I only want to apply this to one specific application. Can you please explain what I'm missing?
